### PR TITLE
update default bootstrap [January 2021]

### PR DESCRIPTION
**Pull request by brodybits**
(with some security updates)

Thanks again to @jasongin for publishing and supporting this super-awesome project!

### CODE REVIEW

**Comment by zsoltlengyelit** [comment]
hi @jasongin , are you going to merge this pr?

our company likes this tool very much & thanks four your efforts!

**Comment by jasongin** [comment]
Thanks for the reminder. Looking at this now, my only hesitation is that this will significantly increase the initial download size. For some reason, the size of the Node.js distribution package increased by about 50% from v12 to v14. (Windows 10.7 MB -> 17.2 MB, Mac 13.7 MB -> 20.2 MB). It appears to be the `node` binary that increased by a lot. Does anyone know why?

**Comment by jasongin** [comment]
I see the size increase is due to full ICU support: https://github.com/nodejs/help/issues/3114

**Comment by brodybits** [comment]
Aargh! We *could* continue using Node.js 12 until it reaches EOL. I *did* check out `deno` but it seems to be getting large as well: https://github.com/denoland/deno/releases

Any more ideas ... anyone?

File: defaults.json [hunk 1]
```diff
@@ -5,5 +5,5 @@
 		"default": "node",
 		"node": "https://nodejs.org/dist/"
 	},
-	"bootstrap": "node/12.18.4"
+	"bootstrap": "node/14.15.4"
```
Comment (brodybits) [inline defaults.json:8]:
```suggestion
	"bootstrap": "node/12.20.1"
```

Comment (brodybits) [reply defaults.json:8]: Going back to `12.20.1` for now, will do a quick test.

**Comment by brodybits** [comment]
@jasongin I have changed it to use `12.20.1` and tested on both macOS and Windows machines.

From https://nodejs.org/en/about/releases/ it looks like we have until April next year to find another solution.

I hope this can be reviewed and merged now. Thanks again!

**Comment by jasongin** [comment]
Thanks!

**Pull request merged**